### PR TITLE
Bugfix 438/bug bash fixes Closes #438

### LINKED ISSUE #438: BUG> FE bugs from bug bash on 11/4 (S)
Listed bugs to work on
- [x] If an error occurs while adding a member profile, then the team member should not be added to context
- [x] display member location on the card
- [x] remove mandatory from insperity id while adding a member
- [x] remove the term 'member' from edit, delete and terminate buttons on Directory page
- [x] remove the term 'team' from edit and delete buttons on Team page
- [x] change 'Add member' icon to the 'Add team' icon
- [x] add search team, search team on the left add team on the right
- [x] search member on the left add member on the right
- [x] PDL can't add their own note

### PR DESCRIPTION
**Pull request by jahanner**

### CODE REVIEW

**Comment by bhatiam12** [comment]
@jahanner The build is failing 
```./src/components/action_item/ActionItemsPanel.jsx
87
  Line 69:6:  React Hook useEffect has a missing dependency: 'getActionItems'. 
Either include it or remove the dependency array  react-hooks/exhaustive-deps
88

89
```

**Review by meyerinj — APPROVED** [review]

File: web-ui/src/components/action_item/ActionItemsPanel.jsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ const doUpdate = async (actionItem) => {
 
 const updateItem = debounce(doUpdate, 1500);
 
-const ActionItemsPanel = ({ checkinId, memberName }) => {
+const ActionItemsPanel = ({ checkinId, memberName, mockActionItems }) => {
```
Comment (mkimberlin) [inline web-ui/src/components/action_item/ActionItemsPanel.jsx:30]: not gonna lie..I don't like this one bit.

Comment (jahanner) [reply web-ui/src/components/action_item/ActionItemsPanel.jsx:30]: That was an attempt at making the test work, which I still didn't end up fixing, so I can remove it

File: web-ui/src/components/action_item/ActionItemsPanel.test.js [hunk 2]
```diff
@@ -61,16 +61,33 @@ it("renders correctly", () => {
...
+      createdbyid,
+      description: "description",
+      id: "887d-43d8-82b1-decf4bb64926",
+      priority: 2.5,
+    },
+  ];
   const { container } = render(
     <AppContextProvider value={initialState}>
-      <ActionItemsPanel />
+      <ActionItemsPanel memberName="ms. test" mockActionItems={actionItems} />
```
Comment (mkimberlin) [inline web-ui/src/components/action_item/ActionItemsPanel.test.js:87]: I know this is being skipped...but I'd remove the mock stuff from here so someone doesn't bring it back unwittingly.

File: web-ui/src/components/member-directory/MemberSummaryCard.jsx [hunk 3]
```diff
@@ -51,9 +53,9 @@ const MemberSummaryCard = ({ member, index }) => {
         {isAdmin && (
           <div className="member-card-actions">
             <CardActions>
-              <Button onClick={handleOpen}>Edit Member</Button>
-              <Button>Terminate Member</Button>
-              <Button>Delete Member</Button>
+              <Button onClick={handleOpen}>Edit</Button>
+              <Button>Terminate</Button>
+              <Button>Delete</Button>
```
Comment (mkimberlin) [inline web-ui/src/components/member-directory/MemberSummaryCard.jsx:58]: Let's convert these to a split button: https://material-ui.com/components/button-group/#split-button

(can make another story)

Comment (jahanner) [reply web-ui/src/components/member-directory/MemberSummaryCard.jsx:58]: I'll make a note for story planning

File: web-ui/src/components/team-results/TeamResults.css [hunk 4]
```diff
@@ -1,3 +1,29 @@
 .MuiCard-root {
-    margin-bottom: 1rem;
+  margin-bottom: 1rem;
+}
+
+.summary-card {
+  max-height: 15rem;
+  max-width: 45%;
+  min-width: 45%;
```
Comment (mkimberlin) [inline web-ui/src/components/team-results/TeamResults.css:8]: pfffft

File: web-ui/src/components/team-results/TeamResults.jsx [hunk 5]
```diff
@@ -1,49 +1,73 @@
...
-        name: PropTypes.string,
-        description: PropTypes.string
-    }))
+  teams: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      name: PropTypes.string,
+      description: PropTypes.string,
+    })
+  ),
```
Comment (mkimberlin) [inline web-ui/src/components/team-results/TeamResults.jsx:20]: False advertising...

**Review by mkimberlin — APPROVED** [review]

File: web-ui/src/components/team-results/TeamResults.css (hunk 4 above)
Comment (jahanner) [reply web-ui/src/components/team-results/TeamResults.css:8]: Hey you wanted them to look like that lol

**Pull request merged**